exo/inference/tinygrad/models/llama.py

```
@@ -48,6 +48,56 @@ def repeat_kv(x: Tensor, n_rep: int) -> Tensor:
+
+    # Reshape weights and input for batched operations
+    w_group = self.weight.reshape(*self.scales.shape, group_size).permute(1, 0, 2).cast(dtype=dtypes.float)
+    x_group = x_flat.reshape(total_batch, n_groups, group_size).permute(1, 0, 2)  # [n_groups, total_batch, group_size]
+
+    term1 = (x_group.dot(w_group.permute(0, 2, 1)))*self.scales.T.reshape(n_groups, 1, out_features)  # [n_groups, 1, out_features]
+    term2 = x_group.sum(axis=-1, keepdim=True)*self.biases.T.reshape(n_groups, 1, out_features)  # [n_groups, total_batch, out_features]
+    return (term1 + term2).sum(axis=0).reshape(*orig_shape[:-1], out_features).cast(dtype=dtypes.half)
+
+
+class AffineQuantizedEmbedding:
+  def __init__(self, vocab_size: int, embed_size: int, group_size: int = 64):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)